config/environments/test.rb

```diff
@@ -28,9 +28,6 @@
   config.consider_all_requests_local       = true
   config.action_controller.perform_caching = false
 
-  # Raise exceptions instead of rendering exception templates
-  config.action_dispatch.show_exceptions = false
-
```

Comment:
It is fine to leave it this way, I just added a comment in the test incase someone changes the setting back for some reason.